presto-spi/src/main/java/com/facebook/presto/spi/type/CharType.java

```diff
@@ -30,6 +30,7 @@
         extends AbstractVariableWidthType
 {
     public static final int MAX_LENGTH = 65_536;
+    public static final CharType CHAR = new CharType(MAX_LENGTH);
```

Comment:
I think you use this char constant only in `TestJdbcQueryBuilder`. If it's only used there, then maybe adding it to SPI doesn't make much sense now? When writing char code, I explicitly tried not to add code dependency to static CHAR type instance, which are hard to untangle later on. 

Char without length specified has length of 1, but I would avoid depending on it in code base, where it should be irrelevant. Varchar has unbounded static type instance specified, but it is legacy code.